src/twitter.d.ts

```diff
@@ -1,11 +1,17 @@
 import { CredentialsArgs } from './Credentials';
-import { TwitterStream } from './TwitterStream';
+import { StreamOptions, TwitterStream } from './TwitterStream';
 
 declare module 'twitter-v2' {
   export interface RequestParameters {
     [key: string]: string | Array<string> | RequestParameters;
   }
 
+  export interface StreamOptions {
+    // Number of seconds to wait for data or a heartbeat ping from Twitter before
+    // considering the stream closed (default value is 30 seconds).
+    timeout: number | undefined;
+  }
+
```

Comment:
You've imported this interface on line 2. This second declaration shouldn't be needed.